less_1.1/src/App.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const todolist = [
+  {
+    id:"1",
+    title:"complete assignment",
+  },
+
+  {
+    id:"2",
+    title:"check assignment well",
+  },
+  {
+    id:"3",
+    title:"submit the assignment ",
+  },
+    ];
```

Comment:
closing square bracket should be aligned all the way to the beginning of the line, no indentation.